tests/DataSet/PHP81/ObjectDataSet81Test.php

```diff
@@ -28,7 +30,9 @@ public function testCollectRules(object $object, array $expectedRules): void
     {
         $dataSet = new ObjectDataSet($object);
 
-        $this->assertEquals($expectedRules, $dataSet->getRules());
+        $arrayRules = $this->toNestedArray($dataSet->getRules());
```

Comment:
Grammatically these would be better options - `$arrayOfRules` / `$rulesArray` / `$rulesArr`. Or maybe `$actualRules`?